app/models/project_step.rb

```diff
@@ -358,10 +367,22 @@ def max_descendant_group_depth
 
   def validate_scheduled_start_date
     if schedule_parent && display_start_date != schedule_parent.dependent_step_start_date
```

Comment:
`display_start_date` should be replaced with `scheduled_start_date`.

Reply:
`dependent_step_start_date` uses `display_end_date` which has also been tweaked for this story

will update specs

Reply:
```
def dependent_step_start_date
    display_end_date.try(:+, 1)
  end
```

We should change this to:

```
  def dependent_step_start_date
    scheduled_end_date.try(:+, 1)
  end
```